consensus/ConsensusFileGenerator.py

```diff
@@ -21,9 +21,14 @@ def __init__(self, data, tables):
         comments_table = tables['comments_table']
         self.consensus_data = data['consensus_data']
         self.lab_classifications = data['lab_classifications']
-        self.history = data['history']
+        self.history = data['history']['history']
```

Comment:
what is the history of the history?

Reply:
This is the history as it was before, the correct. For this fix we had to add an "alternate" history. Variants that have been mapped incorrectly received an incorrect ID. Their cDNA and transcript however are not altered by us, so we can still recognise the variants using these two columns. Those variants are kept in the "alternative history", so these variants have an incorrect id, but can still be recognised using their transcript and cDNA.